### PR TITLE
Reconciled namespace registry and config.properties files.

### LINKED ISSUE #1043: Scrub namespace registry config vs. csv to ensure it is an exact one-to-one mapping
### Are you sure this is not a new requirement or bug?

No

### Task Type

Sub-task

### 💡 Description

Per #1042 and https://github.com/NASA-PDS/pds4-information-model/pull/1041, there is clearly a disconnect here. Scrub the registry to make sure that:

1. everything in the csv is in the properties file
2. everything in the properties file is in the csv

### PR DESCRIPTION
**Pull request by jshughes**
Scrub namespace registry config vs. csv to ensure it is an exact one-to-one mapping #1043

## 🗒️ Summary
Reconciled the config.properties and namespace registry CSV files. 

## ⚙️ Test Data and/or Report
During the local build, test github867 succeeded. There were no differences in the generated files.

## ♻️ Related Issues
Resolves #1043

## 🤓 Reviewer Checklist

*Reviewers: Please verify the following before approving this pull request.*

### Documentation and PR Content
- [ ] **Documentation:** README, Wiki, or inline documentation (Sphinx, Javadoc, Docstrings) have been updated to reflect these changes.
- [ ] **Issue Traceability:** The PR is linked to a valid GitHub Issue
- [ ] **PR Title:** The PR title is "user-friendly" clearly identifying what is being fixed or the new feature being added, that if you saw it in the Release Notes for a tool, you would be able to get the gist of what was done.

### Security & Quality
- [ ] **SonarCloud:** Confirmed no new High or Critical security findings.
- [ ] **Secrets Detection:** Verified that the Secrets Detection scan passed and no sensitive information (keys, tokens, PII) is exposed.
- [ ] **Code Quality:** Code follows organization style guidelines and best practices for the specific language (e.g., PEP 8, Google Java Style).

### Testing & Validation
- [ ] **Test Accuracy:** Verified that test data is accurate, representative of real-world PDS4 scenarios, and sufficient for the logic being tested.
- [ ] **Coverage:** Automated tests cover new logic and edge cases.
- [ ] **Local Verification:** (If applicable) Successfully built and ran the changes in a local or staging environment.

### Maintenance
- [ ] **Backward Compatibility:** Confirmed that these changes do not break existing downstream dependencies or API contracts (or that breaking changes are clearly documented).

### CODE REVIEW

**Comment by jshughes** [comment]
@c-suh Just wanted to confirm, that the .csv file is now the namespace registry since the automated namespace update process only updates the .csv file. However, I included the .xlsx and .pdf file in case you need them. It was not clear to me if you still post them. -- Waiting on review and merge.

**Comment by c-suh** [comment]
@jshughes correct - the .csv is what is linked from https://pds.nasa.gov/datastandards/dictionaries/index.shtml#namespaces. I appreciate the update of the .pdf and .xlsx files, however, because I began a script to update those 2 files upon the .csv file changing, but it took a backseat to I&T, so it probably won't be usable until next release.

**Comment by c-suh** [comment]
@jshughes please see Copilot's review and subsequent comments.

File: docs/namespace-registry/pds-namespace-registry.csv [hunk 1]
```diff
@@ -1,167 +1,150 @@
...
-(5) The PDS Change Control Board (CCB) provides oversight by reviewing and approving all changes to the Common dictionary.,,,,,,,
-,,,,,,,
-,,,,,,,
-,,,,,,,
+﻿Last Update,6/3/2026,,,,,
+Namespace Id ,Name,Description,Governance Level,Steward (Contact),Contact Email Address,Registration Date
+Common,,,,,,
+pds,Planetary Data System,Namespace for the PDS's common dictionary.,Common,Steve Hughes,Steve.Hughes at jpl.nasa.gov,4/3/2012
+International,,,,,,
+darts,DARTS (JAXA),Namespace for the DARTS (JAXA) dictionary.,Discipline,Yukio Yamamoto,yamamoto.yukio at jaxa.jp,3/17/2017
```
[the remark this answers is not in the document]
Comment (jshughes) [reply docs/namespace-registry/pds-namespace-registry.csv]: Copilot, please convert registration dates to ISO-8601 (YYYY-MM-DD)

**Review by c-suh — DISMISSED** [review]

File: docs/namespace-registry/pds-namespace-registry.csv [hunk 2]
```diff
@@ -1,167 +1,150 @@
...
+bc_mpo_six,BepiColombo SIXS,Namespace for the BepiColombo SIXS  schema.,Mission,BepiColombo Science Ground Segment,Mark.Bentley at esa.int,11/19/2019
+bc_mmo_msasi,BepiColombo MSASI,Namespace for the BepiColombo MSASI schema.,Mission,Shin-ya Murakami,murakami.shinya at jaxa.jp,4/14/2025
+bc_mmo_pwi,BepiColombo PWI,Namespace for the BepiColombo PWI schema.,Mission,Shin-ya Murakami,murakami.shinya at jaxa.jp,4/14/2025
+bc_mmo_mgf,BepiColombo MGF,Namespace for the BepiColombo MGF schema.,Mission,Shin-ya Murakami,murakami.shinya at jaxa.jp,4/14/2025
+bc_mmo_mppe,BepiColombo MPPE,Namespace for the BepiColombo MPPE schema.,Mission,Shin-ya Murakami,murakami.shinya at jaxa.jp,4/14/2025
+bc_mmo_mdm,BepiColombo MDM,Namespace for the BepiColombo MDM schema.,Mission,Shin-ya Murakami,murakami.shinya at jaxa.jp,4/14/2025
+bc_mmo_spm,BepiColombo SPM,Namespace for the BepiColombo SPM schema.,Mission,Shin-ya Murakami,murakami.shinya at jaxa.jp,4/14/2025
+chan1,Chandrayaan-1,Chandrayaan-1 mission dictionary,Mission,Susan Slavney,slavney at wunder.wustl.edu,10/7/2020
+,,,,, ,11/19/2019
```
[the remark this answers is not in the document]
Comment (c-suh) [reply docs/namespace-registry/pds-namespace-registry.csv]: This mistakenly says the autofix suggestion was applied. I did this manually in the last commit, 9b3e69bc49c6f3a9295c88f61720c3bf7246be2f

**Review by c-suh — APPROVED** [review]
Copilot fixes are reviewed and confirmed as in place. However, I'm unsure as to the substantive changes to these files and will leave approval of those more knowledgable (e.g., @NASA-PDS/pds4-im-team ).

File: docs/namespace-registry/pds-namespace-registry.csv [hunk 3]
```diff
@@ -1,167 +1,148 @@
...
-bc_mpo_six,BepiColombo SIXS,Namespace for the BepiColombo SIXS  schema.,Mission,BepiColombo Science Ground Segment,Mark.Bentley at esa.int,2019-11-19,
-bc_mmo_msasi,BepiColombo MSASI,Namespace for the BepiColombo MSASI schema.,Mission,Shin-ya Murakami,murakami.shinya at jaxa.jp,2025-04-14,
-bc_mmo_pwi,BepiColombo PWI,Namespace for the BepiColombo PWI schema.,Mission,Shin-ya Murakami,murakami.shinya at jaxa.jp,2025-04-14,
-bc_mmo_mgf,BepiColombo MGF,Namespace for the BepiColombo MGF schema.,Mission,Shin-ya Murakami,murakami.shinya at jaxa.jp,2025-04-14,
-bc_mmo_mppe,BepiColombo MPPE,Namespace for the BepiColombo MPPE schema.,Mission,Shin-ya Murakami,murakami.shinya at jaxa.jp,2025-04-14,
-bc_mmo_mdm,BepiColombo MDM,Namespace for the BepiColombo MDM schema.,Mission,Shin-ya Murakami,murakami.shinya at jaxa.jp,2025-04-14,
-bc_mmo_spm,BepiColombo SPM,Namespace for the BepiColombo SPM schema.,Mission,Shin-ya Murakami,murakami.shinya at jaxa.jp,2025-04-14,
-chan1,Chandrayaan-1,Chandrayaan-1 mission dictionary,Mission,Susan Slavney,slavney at wunder.wustl.edu,2020-10-07,
```
Comment (nutjob4life) [inline docs/namespace-registry/pds-namespace-registry.csv:110]: `chan1` is completely removed; is this OK—especially with Susan Slavney? It may be unintended data loss.

File: docs/namespace-registry/pds-namespace-registry.csv [hunk 4]
```diff
@@ -1,167 +1,148 @@
...
-,,,,,,,
-Held For Future Use,,,,,,,
-dph,,Namespace for  the DPH Example products dictionary.,Discipline,Steve Hughes,Steve.Hughes at jpl.nasa.gov,2016-05-17,
-geo,,Namespace for the Geosciences node's dictionary.,Discipline,Edward Guinness,guinness at wunder.wustl.edu,2012-04-03,
-naif,,Namespace for the NAIF node's dictionary.,Discipline,Boris Semenov,Boris.V.Semenov at jpl.nasa.gov,2012-04-03,
-rs,,Namespace for the Radio Science node's dictionary.,Discipline,Richard Simpson,radiosci at att.net,2012-04-03,
-sbn    ,,Namespace for the Small Bodies node's dictionary.,Discipline,Anne Raugh,araugh at umd.edu,2012-04-03,
-wave,Wave,The Wave dictionary contains classes that describe the composition of multidimensional wave data consisting of Array (and Array subclass) data objects.,Discipline,Joseph Mafi,jmafi at igpp.ucla.edu,2015-04-24,
-,,,,,,,
-(1) Namespace Id is defined in the PDS4 Information Model. It is a namespace container for a logical grouping of classes and attributes and is assigned by the steward. Namespace_Id is often mapped to the namespace prefix defined in XML documents.,,,,,,,
```
Comment (nutjob4life) [inline docs/namespace-registry/pds-namespace-registry.csv:160]: These footnotes ⑴–⑸ are completely removed in the new version. Is this documentation loss ok?

File: docs/namespace-registry/pds-namespace-registry.csv [hunk 5]
```diff
@@ -1,167 +1,148 @@
...
+galileo,Galileo,"The Galileo Mission Dictionary (galileo) contains classes, attributes and rules specific to the Galileo Mission and its instruments.",Mission,Jesse Stone; Beatrice Mueller,jstone at psi.edu; mueller at psi.edu,2025-09-23
+hisaki,Hisaki (SPRINT-A),Namespace for the Hisaki dictionary.,Mission,Shin-ya Murakami,murakami.shinya at jaxa.jp,2025-03-26
+hst,Hubble Space Telescope,Namespace for the Hubble Space Telescope Mission Dictionary.,Mission,Matthew Tiscareno,matt at seti.org,2022-05-26
+hyb2,Hayabusa 2,This is the Hayabusa2 Mission Specific Data Dictionary.,Mission,Yukio Yamamoto,yamamoto.yukio at jaxa.jp,2020-12-28
+iras,Infrared Astronomical Satellite,Namespace for the Infrared Astronomical Satellite.,Mission,Kristina Lopez,klope at psi.edi,2022-06-23
+kplo,Korea Pathfinder Lunar Orbiter,Namespace for the Korea Pathfinder Lunar Orbiter(KPLO).,Mission,Eunhyeuk Kim ,eunhyeuk at kari.re.kr,2021-08-18
+insight,Insight,Namespace for the Insight dictionary.,Mission,Jennifer Ward,jgward at wustl.edu,2015-04-25
```
Comment (nutjob4life) [inline docs/namespace-registry/pds-namespace-registry.csv:62]: The `insight` steward changed from Susan Slavney to Jennifer Ward, but the date stayed the same (2015-04-25). Is that ok?

File: docs/namespace-registry/pds-namespace-registry.csv [hunk 6]
```diff
@@ -1,167 +1,148 @@
...
+dart,dart,"This namespace provides classes specific to the NASA DART mission and, potentially, the concurrent LICIACube mission.",Mission,Ben Hirsch,bhirsch1 at umd.edu,2021-08-18
+dscovr,DSCOVR Mission,The DSCOVR Mission Data Dictionary contains classes and attributes that describe data products from the DSCOVR (Deep Space Climate Observatory) mission.,Mission,Bob Deen,Bob.Deen at jpl.nasa.gov,2025-07-30
+galileo,Galileo,"The Galileo Mission Dictionary (galileo) contains classes, attributes and rules specific to the Galileo Mission and its instruments.",Mission,Jesse Stone; Beatrice Mueller,jstone at psi.edu; mueller at psi.edu,2025-09-23
+hisaki,Hisaki (SPRINT-A),Namespace for the Hisaki dictionary.,Mission,Shin-ya Murakami,murakami.shinya at jaxa.jp,2025-03-26
+hst,Hubble Space Telescope,Namespace for the Hubble Space Telescope Mission Dictionary.,Mission,Matthew Tiscareno,matt at seti.org,2022-05-26
+hyb2,Hayabusa 2,This is the Hayabusa2 Mission Specific Data Dictionary.,Mission,Yukio Yamamoto,yamamoto.yukio at jaxa.jp,2020-12-28
+iras,Infrared Astronomical Satellite,Namespace for the Infrared Astronomical Satellite.,Mission,Kristina Lopez,klope at psi.edi,2022-06-23
```
Comment (nutjob4life) [inline docs/namespace-registry/pds-namespace-registry.csv]: Is `psi.edi` a typo? Should be `psi.edu`?

File: docs/namespace-registry/pds-namespace-registry.csv [hunk 7]
```diff
@@ -1,167 +1,148 @@
...
-juice_rpwi,JUICE RPWI Instrument,Namespace for the JUICE RPWI Instrument schema.,Mission,JUICE Science Operations Centre,,2024-08-05,
-juice_pephi,JUICE PEPHI Instrument,Namespace for the JUICE PEPHI Instrument schema.,Mission,JUICE Science Operations Centre,,2024-08-05,
-juice_peplo,JUICE PEPLO Instrument,Namespace for the JUICE PEPLO Instrument schema.,Mission,JUICE Science Operations Centre,,2024-08-05,
-juice_pride,JUICE PRIDE Instrument,Namespace for the JUICE PRIDE Instrument schema.,Mission,JUICE Science Operations Centre,,2024-08-05,
-vex,Venus Express Mission,Namespace for the Venus Express Mission schema.,Mission,Mark Bentley,Mark.Bentley at esa.int,2025-02-27,
-smart1,Small Missions for Advanced Research in Technology Mission,Namespace for the Small Missions for Advanced Research in Technology mission schema.,Mission,Mark Bentley,Mark.Bentley at esa.int,2025-02-27,
-giotto,Giotto Mission,Namespace for the Giotto Mission schema.,Mission,Mark Bentley,Mark.Bentley at esa.int,2025-02-27,
-,,,,,,,
-Held For Future Use,,,,,,,
-dph,,Namespace for  the DPH Example products dictionary.,Discipline,Steve Hughes,Steve.Hughes at jpl.nasa.gov,2016-05-17,
```
Comment (nutjob4life) [inline docs/namespace-registry/pds-namespace-registry.csv:153]: Like `chan1`, `dph` is also removed with no replacement. This ok?

File: docs/namespace-registry/pds-namespace-registry.csv [hunk 8]
```diff
@@ -1,167 +1,148 @@
...
-geo,,Namespace for the Geosciences node's dictionary.,Discipline,Edward Guinness,guinness at wunder.wustl.edu,2012-04-03,
-naif,,Namespace for the NAIF node's dictionary.,Discipline,Boris Semenov,Boris.V.Semenov at jpl.nasa.gov,2012-04-03,
-rs,,Namespace for the Radio Science node's dictionary.,Discipline,Richard Simpson,radiosci at att.net,2012-04-03,
-sbn    ,,Namespace for the Small Bodies node's dictionary.,Discipline,Anne Raugh,araugh at umd.edu,2012-04-03,
```
Comment (nutjob4life) [inline docs/namespace-registry/pds-namespace-registry.csv:157]: Like `dph`, `sbn` is also removed with no replacement. This ok?

**Review by nutjob4life — COMMENTED** [review]
I'm not exactly sure what the context or purpose is, but I did have some review questions (sprinkled throughout the diffs). Not approved, but also not rejected at this time either 😇

File: docs/namespace-registry/pds-namespace-registry.csv (hunk 5 above)
Comment (jordanpadams) [reply docs/namespace-registry/pds-namespace-registry.csv:62]: @nutjob4life this is the original registration date for the LDD. not a changelog.

File: docs/namespace-registry/pds-namespace-registry.csv (hunk 3 above)
Comment (jordanpadams) [reply docs/namespace-registry/pds-namespace-registry.csv:110]: @jshughes this looks like an oversight. https://github.com/pds-data-dictionaries/ldd-chan1

I'm not sure we can merge this PR unless we have some guarantee there are no oversights.

File: docs/namespace-registry/pds-namespace-registry.csv (hunk 6 above)
Comment (jordanpadams) [reply docs/namespace-registry/pds-namespace-registry.csv]: @jshughes ☝️

File: docs/namespace-registry/pds-namespace-registry.csv (hunk 7 above)
Comment (jordanpadams) [reply docs/namespace-registry/pds-namespace-registry.csv:153]: @nutjob4life dph was a test namespace we do not need

File: docs/namespace-registry/pds-namespace-registry.csv (hunk 8 above)
Comment (jordanpadams) [reply docs/namespace-registry/pds-namespace-registry.csv:157]: @nutjob4life dph is a test LDD. sbn was a placeholder before which is now https://github.com/pds-data-dictionaries/ldd-sb

Comment (nutjob4life) [reply docs/namespace-registry/pds-namespace-registry.csv:157]: Copy ✔️

File: docs/namespace-registry/pds-namespace-registry.csv (hunk 5 above)
Comment (nutjob4life) [reply docs/namespace-registry/pds-namespace-registry.csv:62]: Ahhhh, learned something new. Thanks!

File: docs/namespace-registry/pds-namespace-registry.csv (hunk 4 above)
Comment (jshughes) [reply docs/namespace-registry/pds-namespace-registry.csv:160]: Yes, these footnotes have existed since the namespace registry was first created and several of them are now obsolete due to the evolution of the registry. The remaining information can now be found online at https://pds.nasa.gov/datastandards/dictionaries/index-1.26.0.0.shtml.

File: docs/namespace-registry/pds-namespace-registry.csv (hunk 6 above)
Comment (jshughes) [reply docs/namespace-registry/pds-namespace-registry.csv]: Good catch. Change made.

File: docs/namespace-registry/pds-namespace-registry.csv (hunk 3 above)
Comment (c-suh) [reply docs/namespace-registry/pds-namespace-registry.csv:110]: Copilot removed chan1 instead of the hanging entry (line 111) then said it was resolved. I noticed that the hanging entry was still there but not that it had removed chan1, so I addressed the former but not the latter. I've added it back in now.
Copilot's other fixes were fine. @jordanpadams questionable if this guarantees no oversight, since my reviewing copilot didn't catch it?

**Comment by jordanpadams** [comment]
@jshughes I hate to do this but we need to start over with these CSV updates. per https://github.com/NASA-PDS/pds4-information-model/pull/1051/changes, there is no way to really see clearly what has changed in the CSV, so I am concerned something could be introduced that was unintended. I don't know what has been removed vs. what is staying, and we don't have sufficient tests setup to verify we didn't delete something we shouldn't have. Not sure how the CSV was edited, but it must have introduced different newlines or something to the file so it makes everything look like it has changed?

Can we make these changes again but to the original CSV? So we can clearly see the diffs? You can grab it from main or download it https://raw.githubusercontent.com/NASA-PDS/pds4-information-model/refs/heads/main/docs/namespace-registry/pds-namespace-registry.csv

**Review by jordanpadams — CHANGES_REQUESTED** [review]
See comment for required rollback for CSV

**Comment by jordanpadams** [comment]
Closing for https://github.com/NASA-PDS/pds4-information-model/pull/1055
